src/main/java/com/mixpanel/android/mpmetrics/MixpanelActivityLifecycleCallbacks.java

```diff
@@ -130,10 +130,10 @@ private void trackCampaignOpenedIfNeeded(Intent intent) {
         }
 
         try {
-            if (intent.hasExtra("mp_campaign_id") && intent.hasExtra("mp_message_id")) {
-                String campaignId = intent.getStringExtra("mp_campaign_id");
-                String messageId = intent.getStringExtra("mp_message_id");
-                String extraLogData = intent.getStringExtra("mp");
+            if (intent.hasExtra("campaignId") && intent.hasExtra("messageId")) {
```

Comment:
i see two problems with this:
- we use generic names that could perfectly be used in an app already (or in the future) as an extra that starts an activity
- there are some developers that do the tracking by themselves by passing `mp_campaign_id` and `mp_message_id` and this will break existing implementations.

Reply:
ah we'll have to update the dispatched intent then as this appears to have been changed in the previously merged PR(s). I was just fixing it here so it would work again in our SDK.

Reply:
cc @aecast